src/components/PostsList.tsx

```diff
@@ -3,52 +3,55 @@
 import classNames from 'classnames';
 import React from 'react';
 import { Post } from '../types/Post';
+import { useAppDispatch } from '../app/hooks';
+import { setSelectedPost } from '../features/selectedPost/selectedPostSlice';
 
 type Props = {
   posts: Post[];
   selectedPostId?: number;
-  onPostSelected: (post: Post | null) => void;
 };
 
-export const PostsList: React.FC<Props> = ({
-  posts,
-  selectedPostId = 0,
-  onPostSelected,
-}) => (
-  <div data-cy="PostsList">
-    <p className="title">Posts:</p>
+export const PostsList: React.FC<Props> = ({ posts, selectedPostId = 0 }) => {
```

Comment:
```suggestion
export const PostsList: React.FC<Props> = ({ posts, selectedPostId }) => {
```
default value is redundant here